Lib/test/clinic.test.c

```diff
@@ -5520,6 +5520,7 @@ test_deprecate_positional_pos1_len1_optional(PyObject *module, PyObject *const *
     PyObject *a;
     PyObject *b = Py_None;
 
+    // Emit compiler warnings when we get to Python 3.14.
```

Comment:
Whatever we decide w.r.t. the larger feature, the addition of these comments seems like a nice enhancement by itself